apphelpers/contextHelper.go

```diff
@@ -53,19 +50,53 @@ func NewContext(name string) {
 			Name:   "DSPassword",
 			Prompt: &survey.Password{Message: "Please enter your datasource default password"},
 		},
+		{
+			Name: "Folders",
+			Prompt: &survey.Select{
+				Message: "Do you want to monitor a list of folders or monitor all folders?",
+				Options: []string{"list", "all folders"},
+				Default: "list",
+			},
+		},
 	}
 	err := survey.Ask(behaviorQuestions, &promptAnswers)
 	if err != nil {
 		log.Fatal("Failed to get valid answers to generate a new context")
 	}
 
 	//Set Watched Folders
-	foldersList := strings.Split(promptAnswers.Folders, ",")
-	if len(foldersList) > 0 && foldersList[0] != "" {
-		answers.MonitoredFolders = foldersList
+	if promptAnswers.Folders == "list" {
+		folderAnswer := struct {
+			Folders string
+		}{}
+		var folderListQuestion = []*survey.Question{
+			{
+				Name:   "Folder List",
+				Prompt: &survey.Input{Message: "List the folders you wish to manage (example: folder1,folder2)? (Blank for General)?"},
+			},
+		}
+		err := survey.Ask(folderListQuestion, &folderAnswer)
+		if err != nil {
+			log.Fatal("Failed to get valid answers to generate a new context")
+		}
+		foldersList := strings.Split(folderAnswer.Folders, ",")
+		if len(foldersList) > 0 && foldersList[0] != "" {
+			answers.MonitoredFolders = foldersList
+		} else {
+			answers.MonitoredFolders = []string{"General"}
+		}
 	} else {
-		answers.MonitoredFolders = []string{"General"}
+		answers.MonitoredFolders = []string{}
+		for _, currentConfig := range config.Config().Contexts() {
```

Comment:
This isn't valid.  In this case you should just set the folder list to Empty and ensure the IgnoreFilters flag is set to true.